### ISSUE TITLE
Failure to import umi_methods module 

### ISSUE THREAD

**Issue by rpadmanabhan** [CGATOxford/UMI-tools#144]
I encountered that importing umi_methods will fail since the Cython module _dedup_umi fails to import. A quick fix I found for this was by adding this before the import statement : 

```
## To import Cython modules
import pyximport
pyximport.install()

try:
    from umi_tools._dedup_umi import edit_distance
except:
    from _dedup_umi import edit_distance
```
Not sure if this is restricted to my python version, but thought it might be useful in case some else runs into this.

**Comment by IanSudbery** [comment]
I think we didn't bother with this since any module in umitools that imports umi_methods will have already installed pyximport. I assume you are importing umi_methods directly into something?

From version 0.5, due any time now, we will be switching to distributing pre-cythonised extensions, so the pyximport won't be necessary.

**Comment by rpadmanabhan** [comment]
I was just using the command line to run umitools (the dedup function)  , I was getting a traceback error where in _dedup_umi is not imported in umi_methods. 

It doesnt seem like dedup.py is installing pyximport. 

Anycase looks like all this wont be required for the future. Thank you.

**Comment by TomSmithCGAT** [comment]
This will be resolved in v0.5
